Maybe does not support SD / SDHC Cards as well as USB yet?

Tried on one internal SD card reader, and the SD card was recognized
just like a removable USB stick -- the card reader probably was wired
as USB internally. External USB card readers also work fine.

But this is not always the case: e.g. some other SD cards are handled
by the `mmc_block` (Multimedia Card (MMC) block device driver) module,
e.g. when plugged into:

	15:00.2 SD Host controller: Ricoh Co Ltd R5C822 SD/SDIO/MMC/MS/MSPro Host Adapter (rev 21)

One such non-USB SD card devices we could get debugging info for has
udisks "DriveConnectionInterface" set to `sdio`, a block device name
along the lines of `/dev/mmcblk0`, and is not flagged as removable.

Roadmap
=======

[[!tag todo/code]]

* Modify `liveusb/creator.py` to accept `sdio` as well as `usb`:

	iface = str(dev.Get(device, 'DriveConnectionInterface'))
	if iface != 'usb' and self.opts.force != data['device']:

* Hack liveusb/creator to avoid filtering out non-removable devices
  when they're connected to a `sdio` interface.

* Improve the `switch_drive_to_system_partition` function in
  `liveusb/creator.py`: it currently assumes that the first partition
  is called `${DRIVE}1`, which is not the case for SDIO SD cards, in
  which case it's `${DRIVE}p1`. Therefore, the installation fails.

[[!tag release/2.0]]
